variables.tf

```diff
@@ -94,3 +94,9 @@ variable "lb_ip_name" {
   type        = string
   default     = ""
 }
+
+variable "lb_ssl_policy" {
+  description = "A reference (self-link) to an SSLPolicy that will be associated with the bucket load-balancer (if any). If this variable is not set, a default SSL Policy created & managed by google will be used - which usually follows **COMPATIBLE** profile with **TLS v1.0**."
```

Comment:
Even if it creates a default with COMPATIBLE, we can make sure the COMPATIBLE profile is using only TLS 1.2. We do not want any of our LBs to support 1.* anymore. Please do the necessary.

Reply:
@dibosh Default policies that are created/managed by Google cannot be modified

<img width="909" alt="Screenshot 2021-07-05 at 2 28 24 PM" src="https://user-images.githubusercontent.com/975768/124427132-8e2f4a80-dd9d-11eb-9818-87b6cc493daf.png">

Reply:
Oh, I was proposing, as part of this module, we should create a new default policy with 1.2 as TLS. Not using the googles default one tbh. Because the more LBs we create and expose with TLS 1.*, that's another days security concern or vulnerability report and more work!

Reply:
IMO, we should not create new SSL policy for every LB. The lifecycle of SSL Policy is ideally separate from the LB. In fact, it should be like one project has one SSL policy which defines the projects SSL standard. Then multiple LBs in the project should connect to it. Hence, the recommendation to create one `google_compute_ssl_policy` resource in the project level (as shared in the screenshot above) and any number of LBs will just refer to it (as shared in the screenshot).

Reply:
Checking the console, that's the pattern we have leveraged. Define one strict SSL policy. And all LBs (bucket / vm / gke) refer to that.

Reply:
Okay sounds good.